pom.xml

```diff
@@ -54,10 +54,12 @@
         <connection>scm:git:git://github.com/jenkinsci/vsphere-cloud-plugin.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/jenkinsci/vsphere-cloud-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/vsphere-cloud-plugin</url>
-        <tag>HEAD</tag>
+        <tag>${scmTag}</tag>
     </scm>
 
     <properties>
+        <revision>2.22</revision>
+        <changelist>-SNAPSHOT</changelist>
```

Comment:
These two lines look like they're duplicating/conflicting-with the next two lines.